packages/docs/modules/page-config/blocks/alert/index.vue

```diff
@@ -9,6 +9,10 @@ const props = defineProps({
   color: {
     type: String,
     default: 'info'
+  },
+  outline: {
```

Comment:
Always prefer boolean props to be false by default. So it must be solid true. But it is minor.